### DISCUSSION TITLE
Way better than the obsidian-git plugin

### DISCUSSION THREAD

**Discussion by Zubbbz** [Taitava/obsidian-shellcommands#375] [Offtopic]
I love this plugin! I love having the option to run on certain events, preactions are great for git commit messages, the nuances of the configurations of the notifications and what happens with stdout/stderr, all the variables. This is amazing.

**Comment by Taitava** [comment]
Thank you a lot for your kind feedback! ❤️

And preactions (Prompts) will be even better when I get #365 finished and released. It will allow having different kinds of input fields, e.g. a multiline fields so you can have linebreaks in your commit messages. 😉

**Reply by Taitava** [reply]
Multiline fields in prompts are now available in SC `0.21.0`.
